mimesis/providers/address.py

```diff
@@ -84,7 +87,7 @@ def address(self) -> str:
 
         )
 
-    def state(self, abbr: bool = False) -> str:
+    def state(self, abbr: bool =False) -> str:
```

Comment:
https://www.python.org/dev/peps/pep-3107/#syntax

![](https://i.imgur.com/K57XoKG.png)

Reply:
If that's the convention you'd like to see in this code base, then that's what I'll follow.